Sources/Suggestions/Score.swift

```diff
@@ -26,6 +26,7 @@ extension Score {
     // swiftlint:disable:next cyclomatic_complexity
     init(title: String?, url: URL, visitCount: Int, query: Query, queryTokens: [Query]? = nil) {
         // To optimize, query tokens can be precomputed
+        let query = query.lowercased()
```

Comment:
Probably not strictly required because usually input will be lowercase, but while testing caused some confusion.